client/src/redux/action-types.js

```diff
@@ -16,6 +16,7 @@ export const actionTypes = createTypes(
     'resetUserData',
     'tryToShowDonationModal',
     'executeGA',
+    'saveChallengeComplete',
```

Comment:
```suggestion
```
So when you createAsyncType the Error and Complete  types are created. I think we can remove thi.s

Reply:
Fixed